src/gen/engine/src/main/java/com/paxful/sdk/PxsdkGenerator.java

```diff
@@ -0,0 +1,66 @@
+package com.paxful.sdk;
```

Comment:
That's a bad namespace, if we have JVM sdk, then most likely it will use it. Let's use something else, more accurately reflecting what this class is related to - e.g. com.paxful.jssdk maybe?

Reply:
fixed